python/paddle/tensor/linalg.py

```diff
@@ -29,6 +29,8 @@
 
 __all__ = []
 
+_range = range
+
```

Comment:
This writing method affects the readability of the code. If it is to avoid conflicts with the name of the `range` parameter in `histogramdd`, the `range` parameter can be adjusted to `ranges`